src/clr-core/.storybook/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": []
```

Comment:
Will this remain empty?

Reply:
Yeah, this removes the default presets that storybook applies to our code.